system/jhtree/jhtree.cpp

```diff
@@ -653,14 +686,9 @@ class CNodeMRUCache : public CMRUCacheOf<CKeyIdAndPos, CJHTreeNode, CNodeMapping
         if (((size32_t)-1) == memLimit) return false;
         return sizeInMem > memLimit;
     }
-    virtual void elementAdded(CNodeMapping *mapping)
```

Comment:
This is still needed - it needs to add FIXED_NODE_OVERHEAD.  Alternative (and simpler) is to add that in noteReady() function.  Another alternative is to included that in getMemSize() - which may be better long term (and remove delta from elementRemoved).